src/_includes/steering-committee.html

```diff
@@ -1,54 +1,67 @@
 <div class="container">
   <div class="row">
     <div class="col-sm-4">
-      <h5>Hack Night Lead:</h5>
+      <h5>Hack Night Lead</h5>
       <div>
         <img
           class="steeering-committee-image"
-          alt="Photo of Colin King-Bailey"
-          src="/assets/images/steering-committee/Colin.jpg"
+          alt="OpenOakland logo"
```

Comment:
quick question, did you mean to replace everyone's image on the about-us page with the open oakland logo? It definitely seems intentional but wanted to make sure it wasn't an accidental find/replace.

![image](https://user-images.githubusercontent.com/1066253/107723611-7217f900-6c96-11eb-875c-7d9aeb015edb.png)

Reply:
Yeah, they're placeholder until folks get me their pics. :)